keps/0002-controller-manager.md

```diff
@@ -206,7 +210,7 @@ taints.
 ### API Server Changes
 
 Finally, in the kube-apiserver, the cloud provider is used for transferring SSH keys to all of the nodes, and within an a
-dmission controller for setting labels on persistent volumes.
+admission controller for setting labels on persistent volumes.
```

Comment:
There is an "a" on the line above. After this change it looks like "...within an a admission..."